k8s/dist_test.md

```diff
@@ -0,0 +1,39 @@
+# Testing procedures for distributed zqd
```

Comment:
We use - instead of _ in all other markdown file names, so call this file dist-test.md.